### DISCUSSION TITLE
Expose Lights But Not Camera?

### DISCUSSION THREAD

**Discussion by BananimusPrime** [dgreif/ring#964] [Q&A]
Hi, apologies if I missed this somewhere but I did have a look and couldn't see anything. Is there a way to add my Spotlight Cam's lights to HK while hiding the video feed itself? I'd like to control the lights with automations, but not interested in having the live view in Home.

Thanks!

**Comment by dgreif** [comment]
There is no option for this currently. I'd recommend just putting the camera in a different room in the Home app if you don't want to see the live feed tile
